src/pheval_exomiser/prepare/yaml_to_family_phenopacket.py

```diff
@@ -0,0 +1,275 @@
+import click
+import yaml
+from google.protobuf.json_format import MessageToJson
+from google.protobuf.timestamp_pb2 import Timestamp
+from oaklib.implementations.pronto.pronto_implementation import ProntoImplementation
+from oaklib.resource import OntologyResource
+from phenopackets import (
+    Diagnosis,
+    Family,
+    File,
+    GeneDescriptor,
+    GenomicInterpretation,
+    Individual,
+    Interpretation,
+    MetaData,
+    OntologyClass,
+    Pedigree,
+    Phenopacket,
+    PhenotypicFeature,
+    Resource,
+    VariantInterpretation,
+    VariationDescriptor,
+    VcfRecord,
+)
+from ..utils.file_utils import files_with_suffix
+
+resource = OntologyResource(slug="hp.obo", local=False)
+oi = ProntoImplementation(resource)
+genotype_resource = OntologyResource(slug="geno.owl", local=False)
+go_oi = ProntoImplementation(genotype_resource)
+
+
+class YamlToFamilyPhenopacketConversion:
```

Comment:
What is the purpose of this?

Reply:
This method converts .yaml analysis job files for Exomiser to Phenopackets when given both the yaml file and a "Diagnosis" file which contains the affected gene and variant data for the individual. This was useful for when I was converting .yaml files for different benchmarking sets. 

> @yaseminbridges
> 
> I hope you are cool with spending some time to getting this right, if not, let me know! I would like to iterate fast as well, but at least for the major chunks, we should figure out where the home should be first before merging..
> 
> * [ ]  What is the exact role of src/pheval_exomiser/resources/hgnc_complete_set.txt? Is this exomiser specific?

The hgnc_complete_set.txt is used within this function to retrieve the relevant HGNC:ID for the affected gene. Appeared to be more reliable to retrieve this data locally rather than using the REST API in terms of the possible number of requests being sent